### PR TITLE
majors/minors, edit/submit userinfo, subscribe/unsubscribe from groups adds/removes surveys to homepage

### PR DESCRIPTION
**Pull request by sungbae**

### CODE REVIEW

**Comment by sungbae** [comment]
majors/minors are displayed. fixed the edit/submit userinfo bug. All that is left is for issue #191.

File: models/basemodel.py [hunk 1]
```diff
@@ -275,6 +275,18 @@ def unsubscribe_user(self, user_id, row_id, user_subscription_name=None):
                 logging.warn("row_id {0} not in user {1}".format(row_id, user_subscription_name))
                 pass
             r.db(self.DB).table(user_table).get(user_id).update({user_subscription_name: user_subscription}).run(self.conn)
+        logging.info("ROW_ID IS {0}".format(row_id))
+        active_surveys = r.db(self.DB).table('Group').get(row_id).getField('active_surveys').run(self.conn)
```
Comment (sungbae) [inline models/basemodel.py]: Is it possible to get database information like this? I keep getting an error that says 

`File "/home/user/github/scq/models/basemodel.py", line 279, in unsubscribe_user
        active_surveys = r.db(self.DB).table('Group').get(row_id).getField('active_surveys').run(self.conn)
    AttributeError: 'Get' object has no attribute 'getField'
`

Note: row_id is my group name (i.e. id in Group table). It works when I directly run it in my data explorer for rethinkdb. Why doesn't this recognize getField()?

Comment (nicot) [inline models/basemodel.py]: https://rethinkdb.com/api/python/get_field/

The python API is different from the Javascript API.

Comment (sungbae) [inline models/basemodel.py]: Thanks Nico, I can always count on you.

**Pull request merged**